code/modules/ruins/spaceruin_code/bigderelict1.dm

```diff
@@ -6,3 +6,13 @@
 /obj/item/paper/crumpled/ruins/bigderelict1/coward
 	icon_state = "scrap_bloodied"
 	info = "If anyone finds this, please, don't let my kids know I died a coward.."
+
+/datum/outfit/tradepostcoward
+	name = "derelict tradepost officer"
```

Comment:
this is like the only non capitalized outfit name in the game

Reply:
Thanks for catching this